docs/DafnyRef/UserGuide.md

```diff
@@ -371,8 +371,8 @@ For the formatter to work, the file should be parsed correctly by Dafny.
 
 There are four ways to use the formatter:
 
-* `dafny format file1.dfy file2.dfy ...` formats each file provided in arguments.
-* `dafny format --print:- file1.dfy file2.dfy ...` format each file but instead of writing files, output their formatted content in stdout
+* `dafny format file1.dfy file2.dfy ...` formats each file provided in arguments, altering the files in place.
+* `dafny format --print file1.dfy file2.dfy ...` format each file but instead of altering the files, output the formatted content to stdout
 * `dafny format` Look up for all the Dafny files recursively in the current repository and format them.
 * `dafny format --check` with the same options as above. It will print a message
```

Comment:
Still not clear whether this option changes any files. I vote for no change. Rather print out a list of which files would change, preferably with a diff of the changes.

Reply:
> There is no way to format material from stdin. I suggest `dafny format` take its input from stdin (and prints to stdout by default). And then have a directory required to format files in a directory.

I added support for the option "--stdin" which existed before for Dafny.

Reply:
I refactored this support into a separate PR as well
https://github.com/dafny-lang/dafny/pull/3429